packages/manager/apps/hub/src/catalog/template.html

```diff
@@ -0,0 +1,60 @@
+<h1 data-translate="manager_hub_catalog"></h1>
+<form
+    novalidate
+    class="d-flex justify-content-around align-items-center flex-wrap"
+>
+    <span data-translate="manager_hub_catalog_filter"></span>
+    <oui-field
+        label="{{:: 'manager_hub_catalog_universe' | translate }}"
+        size="m"
+    >
+        <oui-select
+            name="universe"
+            model="$ctrl.selectedUniverses"
+            items="$ctrl.universes"
+            multiple
+            on-change="$ctrl.onUniverseChange($ctrl.selectedUniverses)"
+        >
+        </oui-select>
+    </oui-field>
+    <oui-field
+        label="{{:: 'manager_hub_catalog_category' | translate }}"
+        size="m"
+    >
+        <oui-select
+            name="category"
+            model="$ctrl.selectedCategories"
+            items="$ctrl.categories"
```

Comment:
data-

Reply:
@varun257  it is a custom component, not mandatory